test/integration/scheduler/queue_test.go

```diff
@@ -507,6 +507,56 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			wantRequeuedPods:          sets.New("pod2"),
 			enableSchedulingQueueHint: []bool{true},
 		},
+		{
+			name:         "Pod rejected by the NodeUnschedulable plugin is requeued when the node is turned to ready",
+			initialNodes: []*v1.Node{st.MakeNode().Name("fake-node").Unschedulable(true).Taints([]v1.Taint{{Key: v1.TaintNodeNotReady, Effect: v1.TaintEffectNoSchedule}}).Obj()},
```

Comment:
Taint isn't related to NodeUnschedulable plugin, right? Can we remove Taints from all Nodes in all test cases to make it clear that only .spec.unschedulable is involved in?

Reply:
Yes, I agree. NodeUnschedulable plugins seems related to only pod tolerations spec.

I tried removing all taints from newly added test cases.
The first two cases passed, but the last one failed.
- I haven't figured out the cause of the failure from >4800 lines of log.
- The last case passes if the fake-node2 node has NodeNotReady taint.

Reply:
> but the last one failed.

Can you please just push your implementation (which you removed taints) so that I can see failure test logs in CI?

Reply:
OK, I just pushed it.

Reply:
From locally executed result, the failure is due to expected and actual value mismatch.
Pod1 is unintentionally requeued on unschedulable node creation.

```
    eventhandlers.go:80: I0918 16:55:16.177708] Add event for node node="fake-node2"
    node_tree.go:65: I0918 16:55:16.177766] Added node to NodeTree node="fake-node2" zone=""
    queue_test.go:635: triggered tt.triggerFn, will check if tt.requeuedPods are requeued
    scheduling_queue.go:572: I0918 16:55:16.177876] Pod moved to an internal scheduling queue pod="core-res-enqueueb7404314-f7b6-4fa5-bebe-807a0bbbb3ad/pod1" event="NodeAdd" queue="Active"
    scheduling_queue.go:1102: I0918 16:55:16.177927] Pod moved to an internal scheduling queue pod="core-res-enqueueb7404314-f7b6-4fa5-bebe-807a0bbbb3ad/pod1" event="NodeAdd" queue="Active" hint=<queue.queueingStrategy>: 1
...
    queue_test.go:647: Expect Pods map[] to be requeued, but map[pod1:{}] are requeued actually
```